modules/tinymce/src/themes/silver/main/ts/modes/Inline.ts

```diff
@@ -124,6 +137,18 @@ const render = (editor: Editor, uiComponents: RenderUiComponents, rawUiConfig: R
   };
 };
 
+const getApi = (editor): InlineApi => ({
+  ui: {
+    show: () => {
```

Comment:
Not sure how beneficial this mapping is at the moment but the idea is that we'll extend this API in the future.